client/src/main/scala/drt/client/components/PortDashboardPage.scala

```diff
@@ -20,67 +22,67 @@ object PortDashboardPage {
+              GoogleEventTracker.sendEvent("dashboard", "Switch Period", period.toString)
+              p.router.set(p.dashboardPage.copy(period = Option(period)))
+            }
 
+            <.div(
               <.div(
+                <.div(^.className := "btn-group no-gutters", VdomAttr("data-toggle") := "buttons",
+                  periods.zipWithIndex.map {
+                    case (p, index) => <.div(
+                      ^.className := s"btn btn-primary${if (p == displayPeriod) " active" else ""}",
+                      s"${p.start.prettyTime()}-${p.end.prettyTime()}", ^.onClick ==> switchDashboardPeriod(index)
+                    )
+                  }.toTagMod)),
+              terminals.map { terminalName =>
                 <.div(
-                  <.div(^.className := "btn-group no-gutters", VdomAttr("data-toggle") := "buttons",
-                    periods.zipWithIndex.map {
-                      case (p, index) => <.div(
-                        ^.className := s"btn btn-primary${if (p == displayPeriod) " active" else ""}",
-                        s"${p.start.prettyTime()}-${p.end.prettyTime()}", ^.onClick ==> switchDashboardPeriod(index)
-                      )
-                    }.toTagMod)),
-                terminals.map { terminalName =>
-                  <.div(
-                    <.h3(s"Terminal $terminalName"),
-                    portStateMP().render(portState => {
+                  <.h3(s"Terminal $terminalName"),
+                  portDashboardModel.portState.render(portState => {
+                    portDashboardModel.featureFlags.render(featureFlags => {
                       val portStateForDashboard = portState.windowWithTerminalFilter(displayPeriod.start, displayPeriod.end, portConfig.queues.filterKeys(_ == terminalName))
                       val flightsInTerminal = portStateForDashboard.flights.values.toList
                       val terminalCrunchMinutes = portStateForDashboard.crunchMinutes.values.toList
                       val terminalStaffMinutes = portStateForDashboard.staffMinutes.values.toList
                       val terminalQueuesInOrder = Queues.inOrder(queues.getOrElse(terminalName, Seq()))
 
-                      DashboardTerminalSummary(DashboardTerminalSummary.Props(
-                        flightsInTerminal,
-                        terminalCrunchMinutes,
-                        terminalStaffMinutes,
-                        terminalName,
-                        paxTypeAndQueueOrder(terminalName),
-                        terminalQueuesInOrder,
-                        displayPeriod.start,
-                        displayPeriod.end
-                      ))
+                      DashboardTerminalSummary(DashboardTerminalSummary.Props(flightsInTerminal, terminalCrunchMinutes, terminalStaffMinutes, terminalName, paxTypeAndQueueOrder(terminalName), terminalQueuesInOrder, displayPeriod.start, displayPeriod.end))
```

Comment:
Do you want to split the arguments into multiple lines as it was? I presume this was just intellij being unhelpful

Reply:
👍